packages/rancher-logging/rancher-logging.patch

```diff
@@ -0,0 +1,69 @@
+diff -x '*.tgz' -x '*.lock' -uNr packages/rancher-logging/charts-original/Chart.yaml packages/rancher-logging/charts/Chart.yaml
+--- packages/rancher-logging/charts-original/Chart.yaml
++++ packages/rancher-logging/charts/Chart.yaml
+@@ -1,5 +1,10 @@
+ apiVersion: v1
+ appVersion: 3.4.0
+-description: A Helm chart to install Banzai Cloud logging-operator
+-name: logging-operator
+-version: 3.4.0
++description: A Helm chart to install Banzai Cloud logging-operator and a cluster logging pipeline.
++name: rancher-logging
++version: 3.4.000
++
++annotations:
++  catalog.rancher.io/certified: rancher
++  catalog.rancher.io/namespace: logging-system
```

Comment:
If we are hardcoding these, we may want to hardcode the namespace in other files, as those all use `namespace: {{ .Release.Namespace }}`. But I think for now we leave as is, as when Rancher installs it's all valid, and it's still flexible for non-rancher setup.

Reply:
We chose to not hardcode them in other places for when users install the chart with just helm.